### DISCUSSION TITLE
Column order changes

### DISCUSSION THREAD

**Discussion by hollowheights** [RafaelGB/obsidian-db-folder#881] [General]
I see the columns in the database table change order often, mostly after I delete a column or add new data. I'd strongly prefer to keep both:
-the column order
-the width I set for each column in the table

I'm on Win10 and very excited to use this!

**Comment by RafaelGB** [comment]
hello @hollowheights , looks like a bug since dbfolder should keep both order and width. Could you share your database file or some media to investigate this problem?

Console errors may could helps too. Thanks for use the plugin =)

**Comment by hollowheights** [comment]
Made a test database now and it indeed keeps the order and the width of the columns.
Then tested the problematic database and can't reproduce the error right now, will get back to you after testing it tomorrow again.
Thanks for you work on this!

**Reply by RafaelGB** [reply]
I will be attentive to possible problems, thanks for your feedback!
